configs/gfl/gfl_r50_fpn_1x_coco.py

```diff
@@ -0,0 +1,57 @@
+_base_ = [
+    '../_base_/datasets/coco_detection.py',
+    '../_base_/schedules/schedule_1x.py', '../_base_/default_runtime.py'
+]
+model = dict(
+    type='GFL',
+    pretrained='torchvision://resnet50',
+    backbone=dict(
```

Comment:
The backbone and neck can inherit from RetinaNet in _base_.

Reply:
got it~